rollbar-android/src/main/java/com/rollbar/android/Rollbar.java

```diff
@@ -794,7 +795,7 @@ public void debug(Throwable error, Map<String, Object> custom, String descriptio
   }
 
   /**
-   * Log an error at the level returned by {@link com.rollbar.notifier.Rollbar#level}.
+   * Log an error at the level returned by {@link Level}.
```

Comment:
The previous reference seem to be the correct one, since we are passing `null` here which is going to be replaced with a default by the previously referenced `level` method.